### ISSUE TITLE
Add publication bias methods

### ISSUE THREAD

**Issue by jadahlke** [psychmeta/psychmeta#3]
Add publication-bias analyses for:
- Trim-and-fill
- Egger's test
And other methods, as needed. 

Cross-include the sensitivity_cumulative() and plot_funnel() functions in the to-be-developed publication_bias() function.

**Comment by abblake** [comment]
@jadahlke 

Excellent package, thank you so much! I am just curious, is there a way to extract to ggplot final code from plot_funnel? I'd like to create a funnel plot that has color codes for my publication moderator. Thanks!

**Comment by jadahlke** [comment]
Thank you! The function won’t export the final ggplot code, but ggplot graphics allow for the addition of layers to an object — that’s the beauty of the grammar of graphics. If you want to change color codes, you can do that by extracting the plot from your psychmeta output and adding an aesthetic layer with scale_color_manual() that defines your desired palette.

**Comment by abblake** [comment]
Thank you!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tianx188** [comment]
> Add publication-bias analyses for:
> 
>     * Trim-and-fill
> 
>     * Egger's test
> 
>     * PTES ("Test of Excessive Significance")
>       And other methods, as needed.
> 
> 
> Cross-include the sensitivity_cumulative() and plot_funnel() functions in the to-be-developed publication_bias() function.

@jadahlke I'm looking into doing Trim-and-fill and Egger's test and was wondering when they will be added to psychmeta? Thanks!

**Comment by jadahlke** [comment]
No, not yet. This has not been a priority because these analyses are not unique to psychometric meta-analysis, and we already produce escalc-type output that users can use to run most types of publication bias analyses on their own (e.g., with the `metafor` package).

**Comment by tianx188** [comment]
Got it, thank you!  

Also, for the function plot_cefp(), in the help, it says "Method to calculate standard errors (y-axis). Options are "auto" (default) to use the same method as used to estimate the meta-analysis models, "mean" to calculate SEs using the mean effect size and indivdiual sample sizes, or '"sample"“ to use the SE calculated using the sample effect sizes and sample sizes." 

Wondering if you are able to provide the formulas used for calculating the "auto", "mean" and "sample" SEs?
